### PR TITLE
EWPP-2827: URL change of core site.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

**Review by 22Alexandra — DISMISSED** [review]

**Review by 22Alexandra — DISMISSED** [review]

File: config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+langcode: en
```
Comment (sergepavle) [inline config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml]: Did you change anything in the EU footer config/install? Why do you again import this in the update path?

Comment (sergepavle) [inline config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml]: I've noticed that we have a link like https://ec.europa.eu/commission/index_en which redirects to ec.europa.eu/info site. Should we change this URL as well?

File: tests/features/corporate-blocks.feature [hunk 2]
```diff
@@ -164,8 +164,8 @@ Feature: Corporate blocks feature
   Scenario Outline: The custom footer block shows the correct links in different languages
     Given I am on "the <path> page"
     And the region "ec_footer" contains the links:
-      | Custom Contact            | https://ec.europa.eu/info/contact_en                                 |
-      | Custom Legal notice       | https://ec.europa.eu/info/legal-notice_en                            |
+      | Custom Contact            | https://commission.europa.eu/contact_en                                 |
+      | Custom Legal notice       | https://commission.europa.eu/legal-notice_en                            |
```
Comment (sergepavle) [inline tests/features/corporate-blocks.feature]:
```suggestion
      | Custom Contact            | https://commission.europa.eu/contact_en                               |
      | Custom Legal notice       | https://commission.europa.eu/legal-notice_en                          |
```

File: config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml (hunk 1 above)
Comment (nagyad) [reply config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml]: I don't think we need to change that, I think these redirects will be changed on the server.

Comment (nagyad) [reply config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml]: Right, it's not needed.

**Review by sergepavle — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**